.github/workflows/reusable-integration-tests.yml

```diff
@@ -52,21 +52,24 @@ jobs:
         key: ${{ inputs.php }}-composer-${{ hashFiles('composer.json') }}
 
     - name: Debugging
+      shell: 'bash -x {0}'
```

Comment:
I've opened its output last time and was unable to distinguish the four outputs.